packages/core/src/provisioning.ts

```diff
@@ -314,6 +350,19 @@ export class BasicProvisioner implements Provisioner {
         ]);
     }
 
+    protected _getDefaultAccountProvisioning() {
+        return this.storage.get(AccountProvisioning, "[default]").catch(
+            () =>
+                new AccountProvisioning({
+                    status: this.config.default.status,
+                    statusLabel: this.config.default.statusLabel,
+                    statusMessage: this.config.default.statusMessage,
+                    actionUrl: this.config.default.actionUrl,
+                    actionLabel: this.config.default.actionLabel,
+                })
+        );
+    }
```

Comment:
The default account provisioning is what is returned for an unknown email, i.e. somebody for whom no provisioning status has been set yet. This can now be configured via environment variables or through a database entry with the id `[default]`, which takes precedence. The default provisioning status basically determines whether an unknown user should be allowed to sign up for the service or not, or in other words, whether you need to whitelist or blacklist users. For example, if you want to only allow users that have been whitelisted/provisioned explicitly, the default status should be set to `unprovisioned`. If you want the instance to be open to anyone, the default status should be `active` (which is the default).